music-rpc.js

```diff
@@ -44,6 +45,19 @@ function getProps() {
   });
 }
 
+async function searchSong(props) {
+  const { artist, album } = props;
+  const options = {
+    term: encodeURI(decodeURI(`${artist} ${album}`)),
+    media: 'music',
+    entity: 'album',
+    limit: 1,
+  };
+
+  const result = await itunesAPI.searchItunes(options);
+  return [result.results[0].artworkUrl100, result.results[0].collectionViewUrl];
```

Comment:
>   return [result.results[0].artworkUrl100, result.results[0].collectionViewUrl];
>                             ^
> 
> TypeError: Cannot read properties of undefined (reading 'artworkUrl100')

When no match is found (might happen with imported tracks)

Reply:
fixed